rest-service/manager_rest/test/base_test.py

```diff
@@ -546,14 +546,25 @@ def put_blueprint(self, blueprint_dir, blueprint_file_name, blueprint_id):
                                 blueprint_response['message']))
         return blueprint_response
 
-    def _create_wagon_and_yaml(self, package_name, package_version):
+    def _create_wagon_and_yaml(
+            self,
+            package_name,
+            package_version,
+            package_yaml_file):
```

Comment:
You should probably give `package_yaml_file` a default value to keep backwards compatibility.